serverless/README.md

```diff
@@ -32,17 +34,41 @@ should contain the log state files, e.g.:
 $ export LOG_DIR="/tmp/mylog"
 ```
 
+`sequence` and `client` require the log public key to be provided.
+
+This is supplied by providing the path to the key file using `--public_key` 
+or by setting the `SERVERLESS_LOG_PUBLIC_KEY` environment variable
+
+`integrate` requires the log public and private keys to be provided.
+
+These are supplied by providing the path to the key files using 
+`--public_key` and `--private_key` or by setting the
+ `SERVERLESS_LOG_PUBLIC_KEY` and `SERVERLESS_LOG_PRIVATE_KEY` environment variables.
+
+
+### Generating keys
+To create a new private key pair, usee the `generate_keys` command. Three flags must
+be passed:  
+`--key_name` a name for the signing entity,  
+`--out_pub` path and filename for the public key,  
+`--out_priv` path and filename for the private key
+
+```bash
+$ go run ./serverless/cmd/generate_keys --key_name=astra --out_pub=key.pub --out_priv=key
```

Comment:
There's a slight consistency thing here with the examples - here we create the key files on disk, but below we tacitly assume that the env vars are set, probably worth settling on one mechanism and sticking with it throughout?

Reply:
done, all use key files